lnrpc/rpc.proto

```diff
@@ -336,6 +336,14 @@ service Lightning {
         };
     }
 
+    /** lncli: `subscribechannels`
+    SubscribeChannels creates a uni-directional stream from the server to the
+    client in which any updates relevant to the state of the channels are sent
+    over. Events include new active channels, inactive channels, and closed
+    channels.
+    */
+    rpc SubscribeChannels (ChannelSubscription) returns (stream ChannelUpdate);
```

Comment:
name suggestion: `SubscribeChannelEvents` and `ChannelEventSubscription`.

Reply:
fixed